diff --git a/staticnode/Dockerfile b/staticnode/Dockerfile
index d8782ee..e2ea0af 100644
--- a/staticnode/Dockerfile
+++ b/staticnode/Dockerfile
@@ -1,4 +1,4 @@
-FROM alpine:latest
+FROM ghcr.io/loong64/alpine:latest
 ARG VERSION
 # One of x64 or arm64 or ppc64le
 ARG ARCH
@@ -8,7 +8,7 @@ ARG SHA256SUM
 RUN mkdir -p /build
 WORKDIR /build
 
-RUN apk add --no-cache binutils-gold curl g++ gcc gnupg libgcc linux-headers make python3 libstdc++
+RUN apk add --no-cache curl g++ gcc gnupg libgcc linux-headers make python3 libstdc++
 RUN mkdir -p /out/bin
 
 RUN curl -O https://nodejs.org/dist/$VERSION/node-$VERSION.tar.gz
